repoman/pym/repoman/modules/scan/depend/_depend_checks.py

```diff
@@ -160,7 +161,12 @@ def _depend_checks(ebuild, pkg, portdb, qatracker, repo_metadata):
 			check_slotop(mydepstr, pkg.iuse.is_valid_flag,
 				badsyntax, mytype, qatracker, ebuild.relative_path)
 
+	dedup = collections.defaultdict(set)
 	for m, b in badsyntax:
+		if b in dedup[m]:
```

Comment:
Haha, indeed one of my goals was to make it possible for you to *mostly* neglect the code.